internal/boxcli/usererr/exiterr.go

```diff
@@ -19,22 +19,22 @@ func NewExecError(source error) error {
 	if !errors.As(source, &exitErr) {
 		return source
 	}
-	return &UserExecError{
+	return &ExitError{
 		err: exitErr,
 	}
 }
 
-func (e *UserExecError) Error() string {
+func (e *ExitError) Error() string {
```

Comment:
Not from this PR, but if `ExitError` is just an exec.ExitError you don't need to reimplement all this stuff.